.github/workflows/ci.yml

```diff
@@ -258,16 +258,11 @@ jobs:
           registry: quay.io
           username: ${{ secrets.QUAY_USERNAME }}
           password: ${{ secrets.QUAY_PASSWORD }}
-      - name: "Login to Docker Hub"
-        uses: docker/login-action@v3.1.0
-        with:
-          username: ${{ secrets.DOCKER_USERNAME }}
-          password: ${{ secrets.DOCKER_PASSWORD }}
       - uses: Alfresco/alfresco-build-tools/.github/actions/free-hosted-runner-disk-space@v5.25.0
       - uses: Alfresco/alfresco-build-tools/.github/actions/setup-java-build@v5.25.0
       - name: "Build application"
         run: mvn ${{ env.MAVEN_CLI_OPTS }} clean install -DskipTests -Dproject.revision.key=${{ github.sha }}
-      - name: "Deploy release version to quay.io and Docker Hub"
+      - name: "Deploy release version to quay.io"
         run: bash ./scripts/ci/releaseDockerImage.sh -v ${{ env.RELEASE_VERSION }}
```

Comment:
I'm not sure if the script won't fail due to https://github.com/Alfresco/hxinsight-connector/blob/76c722811098e7903134e496e9276ad129f72d94/scripts/ci/releaseDockerImage.sh#L46 and missing Docker Hub credentials 🤔